.goreleaser.yml

```diff
@@ -209,7 +206,7 @@ brews:
       bin.install "observiq-otel-collector"
       prefix.install "LICENSE", "config.yaml"
       prefix.install Dir["plugins"]
-      prefix.install Dir["storage"]
+      FileUtils.mkdir_p(File.expand_path("~/observiq-otel-collector/storage")) unless File.exists?(File.expand_path("~/observiq-otel-collector/storage"))
```

Comment:
`~/observiq-otel-collector` is not the collector path. Did you test it and it actually worked? I thought the file ops were limited to within the homebrew directory. Does the storage directory have to exist in relation to the collector if so it should be `#{bin.install}/storage`